chain-core/src/init/config.rs

```diff
@@ -72,12 +73,32 @@ impl SlashRatio {
         }
     }
 
+    pub fn get_proportional(
+        self,
+        failing_validators: usize,
+        total_validators: usize,
+    ) -> SlashRatio {
+        let millis =
+            (((3.0 * failing_validators as f64) / total_validators as f64).powi(2) * 1000.0) as u64;
```

Comment:
3.0 can be constant if we need to adjust

Reply:
3 comes from the fact that it can tolerate up to 1/3 of faulting nodes